packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,155 @@
+
+The Android and iOS Identity Engine SDKs represent the sign-in flow as a state machine. You initialize the machine with the details of your Okta Application Integration, request the initial step in the flow, and cycle through responding to steps until either the user is signed in, cancels, or an error occurs.
+
+<div class="common-image-format">
+
+![A diagram showing the sign-in flow.](/img/mobile-sdk/mobile-idx-basic-flow.png "A diagram that shows the sign-in flow.")
+
+</div>
+
+Each sign-in step may include one or more possible user actions, such as:
+
+- Choosing an authenticator, such as Okta Verify or security questions.
+- Entering a One-Time Passcode (OTP).
+- Cancelling the sign-in flow.
+
+## Sign-in objects
+
+Each step in the sign-in flow is represented by a number of different SDK objects:
+
+<div class="common-image-format">
+
+![A diagram showing the SDK objects and their relationships.](/img/mobile-sdk/mobile-idx-objects.png "A diagram that shows the SDK objects for the sign in flow and the relationships between them.")
+
+</div>
+
+- **Response:** The top-level object that represents a step and contains all the other objects. It also indicates a successful sign-in, and is used to send messages for cancelling the sign-in flow, or to retrieve an access token after the sign-in flow succeeds. A response may contain multiple authenticators and remediations.
+- **Remediation:** Represents the main user actions for a step, such as enrolling in an authenticator or entering an OTP. It's also used to update values entered by the user and to request the next step in the flow.
+- **Authenticator:** Represents a type, or factor for verifying the identity of a user, such as a password or OTP.
+- **Method:** Represents a channel for an authenticator, such as using SMS or voice for an authenticator that uses a phone. An authenticator may have zero or more.
+- **Capability:** A user action, that's associated with a remediation, authenticator, or method, such as requesting a new OTP or a password reset.
+- **Field:** Represents an item displayed in the UI, either a static element, such as a label, or a user input, such as a selection list. It also contains state information, such as whether the associated value is required. Options, or lists of choices, are represented by a collection of fields. A field may also contain a form that contains more fields.
+- **Form:** Contains the fields that represent the user action for a remediation.
+- **Configuration:** Contains the settings used by the SDK to connect to an Application Integration in an Okta org.
```

Comment:
Same comment as above, Application Integration is a proper noun that unambiguously identifies something in the Okta org rather than "app" which is ambiguous. "app integration" could refer to the app the developer is writing as its an app integration with the Okta SDK.

Unless there's strong reason to the contrary I'm leaving the text as is.

Reply:
The Infodev style guide suggests using app to refer to apps / app integrations created in the Admin Console and using application to refer to the UI or to formalize a particular 3rd-party application in a process.

This is pretty consistent across dev.okta. 
Also , the only time I can find it used and capitalised is when it is used in the Application Integration Wizard

Reply:
That sounds like a conversation is needed for that bit of the style guide because of the the high likelihood of confusing what "app" or "app integration" is referring to. In the mobile development word "app" means an application on a mobile device and nothing else. In developer documentation, it usually means "the app that you are working on."

It may be that there's more wiggle room in desktop or web development, but there's none in mobile.

The power of a capitalized proper noun is that it makes it absolutely clear what the thing is 😉.

For now I'm going to stick with the capitalized versions... we can always update it later.

Reply:
Probably much like the OTP conversation, this is something that should be discussed and then corrected across the site. Leave it for now and we can come back to it.